modules/hcp-ibm-power-heterogeneous-nodepools-create-agent-cluster.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-heterogeneous-nodepools-create-agent-cluster_{context}"]
+= Create agent cluster
+
+.Procedure
+
+. Create a hosted cluster for heterogeneous node pool
++
+Use a multi-architecture release image when creating the cluster to enable support for heterogeneous node pools. Find the latest multi architecture images on the link:https://multi.ocp.releases.ci.openshift.org/[Multi-arch release images] page.
+
+. Export the cluster namepsace.
++
+[source,terminal]
+----
+$ export CLUSTERS_NAMESPACE=<hosted_cluster_namespace>
+----
+
+. Export the machine classless inter-domain routing (CIDR) notation.
++
+[source,terminal]
+----
+$ export MACHINE_CIDR=192.168.122.0/24
+----
+// Typically the namespace is created by the hypershift-operator 
+// but agent cluster creation generates a capi-provider role that
+// needs the namespace to already exist
+
+. Run the following command to create the hosted control namespace:
++
+[source,terminal]
+----
+$ oc create ns ${HOSTED_CONTROL_PLANE_NAMESPACE}
+----
+
+. Run the following command to create the cluster:
++
+[source,terminal]
+----
+$ hypershift create cluster agent \
+    --name=<hosted_cluster_name> \ <1>
+    --pull-secret=<pull_secret_file> \ <2>
+    --agent-namespace=<hosted_control_plane_namespace> \ <3>
+    --base-domain=<basedomain> \ <4>
+    --api-server-address=api.<hosted_cluster_name>.<basedomain> \
```

Comment:
Annotations with multiline commands require this so they format correctly in the output
```suggestion
$ hypershift create cluster agent \
    --name=<hosted_cluster_name> \// <1>
    --pull-secret=<pull_secret_file> \// <2>
    --agent-namespace=<hosted_control_plane_namespace> \// <3>
    --base-domain=<basedomain> \// <4>
    --api-server-address=api.<hosted_cluster_name>.<basedomain> \
```